build.gradle

```diff
@@ -27,7 +27,7 @@ plugins {
     id 'com.github.johnrengelman.shadow' version '1.2.3'
     id 'com.palantir.configuration-resolver' version '0.1.0'
     id 'com.palantir.git-version' version '0.5.2'
-    id 'org.inferred.processors' version '1.1'
+    id 'org.inferred.processors' version '1.2.4-rc2'
```

Comment:
Why are we using RC build and what repo should that be coming from? Using `1.2.4-rc2` results in the following error for me, while dropping down to `1.2.3` works fine.

```
$ ./gradlew clean idea
:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE

FAILURE: Build failed with an exception.

* Where:
Build file '/Volumes/git/atlasdb/build.gradle' line: 30

* What went wrong:
An exception occurred applying plugin request [id: 'org.inferred.processors', version: '1.2.4-rc2']
> Failed to apply plugin [id 'org.inferred.processors']
   > Could not get unknown property 'idea' for root project 'atlasdb' of type org.gradle.api.Project.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.009 secs
```